### ISSUE TITLE
Using crazy-max/ghaction-import-gpg v3 is not possible to be used without configuration changes

### ISSUE THREAD

**Issue by ingogriebsch** [ingogriebsch/spring-hateoas-siren#102]

**Comment by ingogriebsch** [comment]
The following error occurs if building the project based on v3:

```
Prepare GPG environment0s
    PASSPHRASE: ***
Run crazy-max/ghaction-import-gpg@v3
  with:
    git-user-signingkey: false
    git-commit-gpgsign: false
    git-tag-gpgsign: false
    git-push-gpgsign: false
    workdir: .
  env:
    JAVA_HOME_8.0.265_x64: /opt/hostedtoolcache/jdk/8.0.265/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.265/x64
    JAVA_HOME_8_0_265_X64: /opt/hostedtoolcache/jdk/8.0.265/x64
    GPG_PRIVATE_KEY: ***
    PASSPHRASE: ***
##[error]Input required and not supplied: gpg-private-key
''`

**Comment by ingogriebsch** [comment]
Solved through #192 and #84 and some handmade changes.
